### ISSUE TITLE
feeding stream into network without normalization

### ISSUE THREAD

**Issue by javak87** [seisbench/seisbench#135]
Dear developer,
According to the example that you provided for training phasenet, the data is augmented before feeding to the network. After finishing the training, you used raw stream (without any augmentation like normalization). 
Is it correct procedure or not?
You trained a network with normalized data, but applied for stream data without normalization.
I trained "instance" pre-trained model, for both train and test datasets, the recall and f1score are more than 0.8, but we I used this network for an unfiltered, the number of identified picks drops dramatically.
Best regards,
Javad

**Comment by yetinam** [comment]
Hey @javak87,

thanks for your question! Normalization is automatically performed when using `annotate` or `classify`. The relevant code is [here](https://github.com/seisbench/seisbench/blob/1cdbb52c90d10d4223f53208187beb633d5f84f6/seisbench/models/phasenet.py#L102). However, I admit that it's currently pretty hidden and we should document it somewhere. So the normalization for streams is applied, which allows you to directly feed in the raw stream. However, in training no normalization is applied automatically, so you need to do it manually before. Similarly, if you use different augmentations in training, you might need to adjust the model here.

Regarding the performance on your data, have you checked the characteristic curves output by `annotate`? Maybe you need to lower the detection threshold. In addition, 0.80 F1 score is a bit low than what we saw in other examples, so it's possible that you actually just see the model's performance limitations.

**Comment by yetinam** [comment]
Hey @javak87 ,
quick follow-up: Could you confirm your question is solved and close the issue?

**Comment by javak87** [comment]
Dear yetinam,
Thank you for your follow up.
I used a normal augmentation which is also using in "classify". the result was the ok. After that I used another augmentation in training data which is not common in "classify". In this case, I tried to create a test data set using normal procedure which is existed in seisbench and test the performance of the trained network. In this case the result was not promising. I think maybe the way of calculating F1score that I implemented is not the same as you.
Could you please put an example about calculating F1score to compare the result?
Thanks

**Comment by yetinam** [comment]
Hey @javak87 ,
here is the code block we used in our benchmark to evaluate the detection results: https://github.com/seisbench/pick-benchmark/blob/2fae5941f632fb2146e6bea37f94eb89a2c89bc7/benchmark/collect_results.py#L121
You'll see that we focussed on AUC score rather than F1, but F1 is in there as well. Crucially, F1 scores depend on the threshold you choose, so you will need to optimise this value.

**Comment by javak87** [comment]
Dear yetinam,
Thank you so much for your help.
